src/general.tex

```diff
@@ -6,8 +6,7 @@
 
 \pnum
 Conformance requirements for this document are those defined in subclause
-\ref{general.compliance} in the C++ Standard. Similarly, all references to the C++ Standard in the
-resulting document shall be taken as referring to the resulting document itself.
```

Comment:
Refer to standard documents by their document number, rather than "C++ standard" or "Reflections TS"